platform/windows/display_server_windows.cpp

```diff
@@ -6178,13 +6178,20 @@ DisplayServerWindows::DisplayServerWindows(const String &p_rendering_driver, Win
 
 		bool force_angle = false;
 
-		Vector2i device_id = _get_device_ids(gl_info["name"]);
+		Vector2i device_id;
 		Array device_list = GLOBAL_GET("rendering/gl_compatibility/force_angle_on_devices");
 		for (int i = 0; i < device_list.size(); i++) {
 			const Dictionary &device = device_list[i];
 			if (device.has("vendor") && device.has("name")) {
 				const String &vendor = device["vendor"];
 				const String &name = device["name"];
+				if (device_id == Vector2i() && gl_info["vendor"].operator String().to_upper().contains(vendor.to_upper()) && name.to_upper().begins_with("__ID_CHECK__:")) {
```

Comment:
```suggestion
				if (device_id == Vector2i() && gl_info["vendor"].operator String().containsn(vendor) && name.to_upper().begins_with("__ID_CHECK__:")) {
```
Faster case insensitive check